src/dsp_tools/commands/excel2json/project.py

```diff
@@ -136,6 +203,36 @@ def _create_project_json(
     return overall_success, project
 
 
+def _new_create_project_json(
+    data_model_files: str, onto_folders: list[Path], list_folder: Path | None
+) -> tuple[bool, dict[str, Any]]:
+    overall_success = True
+    lists = None
```

Comment:
This line is not necessary:

```suggestion
```

Reply:
I did it because of a reference before assignement.